CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 ## [Unreleased]
+- Add script to configure apps using workload identity
+- Add script to configure `gcp-compute-persistent-disk-csi-driver-app`
```

Comment:
Why that app has its own entry in the changelog? isn't it included on the first item?

Reply:
It used to be what this PR is about, but we just added everything else